src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -763,7 +763,7 @@ public static void insertConfigIntoConfigFile(RemoteRobot remoteRobot, String fi
                         "Text " + configNameSnippet + " did not appear in the completion suggestion pop-up window",
```

Comment:
Similar to the other comment, I think `configValueSnippet` should be used on lines 762 and 763.